library/src/main/java/com/devbrackets/android/exomedia/core/video/ExoVideoView.java

```diff
@@ -281,7 +281,9 @@ public void onSurfaceTextureAvailable(SurfaceTexture surfaceTexture, int width,
 
         @Override
         public boolean onSurfaceTextureDestroyed(SurfaceTexture surfaceTexture) {
+            Surface surface = emExoPlayer.getSurface();
```

Comment:
If I remember correctly `getSurface()` can return null.  Additionally, how does this interact with releasing the surface texture